e2e/nomostest/clusters/kind.go

```diff
@@ -202,3 +210,45 @@ apiServer:
 	// We failed to create the cluster maxKindTries times, so fail out.
 	return err
 }
+
+var preloadImages = []string{
```

Comment:
nit: I think this could be hard-coded in the Create() func or generated dynamically from testdata with a findDockerhubImages() func or something and then passed as an array to the pullImages and sideLoadImages functions, just for a bit of readability/flexibility. But it's not a big deal.

Reply:
I'd rather this be an explicit singleton shared across threads, since each thread expects the same static images to be sideloaded.